apps/explorer/src/components/module/module-functions-interaction/ModuleFunction.tsx

```diff
@@ -25,6 +25,15 @@ const argsSchema = z.object({
     params: z.array(z.object({ value: z.string().trim().min(1) })),
 });
 
+const connectButtonStyles = cva('!text-bodySmall !rounded-md', {
```

Comment:
FWIW for cases like this, I think it's often cleaner to just use `clsx` inline. CVA is really for component definitions where we want to extract variant props, not for arbitrary application of classes.

Reply:
Done